etc/docker/demo-base/Dockerfile

```diff
@@ -26,7 +27,7 @@ ENV SHELL=/bin/bash \
 ENV HOME=/home/$NB_USER \
     PATH=$ANACONDA_HOME/bin:$HADOOP_PREFIX/bin:$JAVA_HOME/bin:$SPARK_HOME/bin:$PATH
 
-ENV SPARK_VER 2.4.1
+ENV SPARK_VER $SPARK_VERSION
```

Comment:
Is it necessary to set and ENV here?  Just wondering if we can reference the ARG SPARK_VERSION instead and not have SPARK_VER set in the env of the container.

If it is necessary to set the env (SPARK_VER) can we expand it to SPARK_VERSION?

Reply:
This was already there and is following the same pattern as others (such as HADOOP_VER)